dev_guide/deployments/deployment_strategies.adoc

```diff
@@ -73,16 +73,20 @@ The following is an example of the Rolling strategy:
 strategy:
   type: Rolling
   rollingParams:
-    timeoutSeconds: 120 <1>
-    maxSurge: "20%" <2>
-    maxUnavailable: "10%" <3>
-    pre: {} <4>
+    updatePeriodSeconds: 1 <1>
+    intervalSeconds: 1 <2>
+    timeoutSeconds: 120 <3>
+    maxSurge: "20%" <4>
+    maxUnavailable: "10%" <5>
+    pre: {} <6>
     post: {}
 ----
-<1> How long to wait for a scaling event before giving up. Optional; the default is 120.
-<2> `maxSurge` is optional and defaults to `25%` if not specified; see below.
-<3> `maxUnavailable` is optional and defaults to `25%` if not specified; see below.
-<4> `*pre*` and `*post*` are both xref:lifecycle-hooks[lifecycle hooks].
+<1> The time to wait between individual pod updates.
+<2> The time to wait between polling deployment status after update.
+<3> How long to wait for a scaling event before giving up. Optional; the default is 120.
```

Comment:
The automatic rollback does not give up since the only thing involved is the DC controller scaling up the last Complete RC. I've never seen a case where that failed although technically it is possible.

Also we should stop calling RCs as deployments because of naming collision with upstream Deployments.